source/includes/_plans.md

```diff
@@ -115,7 +116,18 @@ Argument | Type | Description
 **interval_unit** | `string` | The frequency that the Subscription acts upon. <br> interval_unit choices: `DAY`, `WEEK`, `MONTH`, `MONTH_END`, `ANNUAL`
 **billing_interval** | `string`| Defines billing frequency. <br> Choices: `WEEK`, `MONTH`, `MONTH_END`
 **currency** | `string`| Three letter ISO currency code as defined by ISO 4217.7
-
+name | `string` | The name of the plan.
+note | `string` | A short description of the plan.
+interval_unit | `string` | The frequency that the subscription acts upon. <br> Choices: `DAY`, `WEEK`, `MONTH`, <br> `MONTH_END`, `ANNUAL`
+interval_count | `integer`| Total number of interval_units.
+billing_interval | `string`| Defines billing frequency. <br> Choices: `WEEK`, `MONTH`, `MONTH_END`
+static | `boolean`| Defines if plan is allowed to be changed.
+items | `array` | List of items associated with Plan. See description below.
+currency | `string` | Three letter ISO currency code as defined by ISO 4217.
+total_amount | `integer` | The amount to be charged on the interval specified. If missing, this will be calculated as the sum of the `items`.
```

Comment:
The amount in the `currency` that will be charged at the end of the billing interval. If missing, the `total_amount` will be calculated as the sum of the `items`.